From 242bc90c398901b4b545b0053af957937f649383 Mon Sep 17 00:00:00 2001
From: atty303 <atty303@users.noreply.github.com>
Date: Tue, 25 Aug 2026 02:55:59 +0900
Subject: [PATCH] winhttp: use 90 second response timeout default

---
 dlls/winhttp/request.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dlls/winhttp/request.c b/dlls/winhttp/request.c
index 60aff3ee4ba..72eabb1d3d2 100644
--- a/dlls/winhttp/request.c
+++ b/dlls/winhttp/request.c
@@ -43,7 +43,7 @@ WINE_DEFAULT_DEBUG_CHANNEL(winhttp);
 
 #define DEFAULT_KEEP_ALIVE_TIMEOUT 30000
 
-#define ACTUAL_DEFAULT_RECEIVE_RESPONSE_TIMEOUT 21000
+#define ACTUAL_DEFAULT_RECEIVE_RESPONSE_TIMEOUT 90000
 
 static int request_receive_response_timeout( struct request *req )
 {
-- 
2.55.0

